example/hatchify-app/frontend/App.tsx

```diff
@@ -11,12 +11,14 @@ export const hatchedReact = hatchifyReact(
   createJsonapiClient("http://localhost:3000/api", schemas),
 )
 
-const TodoList = hatchedReact.components.Todo.Collection
-const TodoColumn = hatchedReact.components.Todo.Column
-const TodoEmptyList = hatchedReact.components.Todo.Empty
-// hatchedReact.model.Todo.findAll({}).then(([todos]) => {
+const TodoList = hatchedReact.components.Admin_Todo.Collection
+const TodoColumn = hatchedReact.components.Admin_Todo.Column
+const TodoEmptyList = hatchedReact.components.Admin_Todo.Empty
+// hatchedReact.model.Admin_Todo.findAll({
+//   fields: { Admin_User: ["name"] },
+// }).then(([todos]) => {
```

Comment:
Are we good to actually remove this commented out code, or are you expecting to use it again?

Reply:
can leave it in for now, will remove before merge to main. i like it to just be able to test `todos[0].` and see if i get the TS hints and didn't break anything :D